### PR TITLE
GTM-SS HTTP Request Tag - Update for v0.2.0

### PR DESCRIPTION
**Pull request by adatzer**

### CODE REVIEW

**Review by paulboocock — APPROVED** [review]
LGTM

**Review by paulboocock — APPROVED** [review]

File: docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md [hunk 1]
```diff
@@ -125,11 +125,68 @@ will result in the following JSON structure:
...
+
+With this setting you can specify a key under which the Snowplow event's unmapped entities will be nested. Alternatively, leaving it blank adds the unmapped entities in the request body without nesting. You can also use dot notation in this value.
+
+#### Snowplow Entities to Add/Edit mapping
+
+Using this table you can specify in each row a specific mapping for a particular context entity. In the columns provided you can specify:
+
+- **Entity Name**: The Entity name to add/edit-mapping (required).¹
+- **Destination Mapped Name**: The key you could like to map it to in the request body (optional: leaving the mapped key blank keeps the same name). You can use dot notation here as well to signify further nesting. This value is independent of the [nesting of unmapped entities](/docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md#nest-all-unmapped-entities-under-key) setting above.
```
Comment (stanch) [inline docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md:151]: for links within the same page you can just use `[...](#nest-all-unmapped-entities-under-key)`

File: docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md [hunk 2]
```diff
@@ -182,3 +241,35 @@ Finally, this section offers two additional configuration options:
...
+- `Do not log`: This option allows you to completely disable logging. No logs will be generated by the Tag.
+- `Log to console during debug and preview`: This option enables logging only in debug and preview containers. This is the **default** option.
+- `Always`: This option enables logging regardless of container mode.
+
+_Note_: Please take into consideration that the logs generated may contain event data.
+
+The logs generated by the HTTP Request GTM-SS Tag are standardized JSON strings.
+The standard log properties are:
+
+```txt
```
Comment (stanch) [inline docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md:260]:
```suggestion
```json
```

[the remark this answers is not in the document]
Comment (adatzer) [reply docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md:260]: probably the comments make it look a bit weird with `json`, 

Screenshot from 2022-09-13 18-49-23

that's why i went for `txt` (i admit it looks too plain though)

[the remark this answers is not in the document]
Comment (adatzer) [reply docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/http-request-tag-for-gtm-ss/http-request-tag-configuration/index.md:260]: My mistake!
Missing colon, fixing asap.

**Pull request merged**